pyls/plugins/jedi_completion.py

```diff
@@ -142,9 +146,9 @@ def _format_completion(d, include_params=True):
         path = path.replace('/', '\\/')
         completion['insertText'] = path
 
-    if (include_params and hasattr(d, 'params') and d.params and
-            not is_exception_class(d.name)):
-        positional_args = [param for param in d.params
+    sig = d.get_signatures()
+    if (include_params and sig and not is_exception_class(d.name)):
+        positional_args = [param for param in sig[0].params
```

Comment:
Why are taking here only the first signature? Also, could you provide an example of an object with multiple signatures?

Reply:
Because PyLS expects only one for completion, but the newer Jedi provides a list of signatures.
https://jedi.readthedocs.io/en/latest/docs/api.html#jedi.Script.get_signatures

I am pretty sure there are cases with multiple signatures for a completion.
https://jedi.readthedocs.io/en/latest/docs/api-classes.html#jedi.api.classes.BaseName.get_signatures
```
    def get_signatures(self):
        """
        Returns all potential signatures for a function or a class. Multiple
        signatures are typical if you use Python stubs with ``@overload``.

        :rtype: list of :class:`BaseSignature`
        """
        return [
            BaseSignature(self._inference_state, s)
            for s in self._get_signatures()
        ]
```